valayDave/tell-me-your-secrets
# Fix regression with saving results

## Proposed change


- I changed the return type from a dict to a class and `DataFrame` didn't like that. This change maps vars over the result to get it back to how it was.

- Bumps version 

## How to test the change


Run over vulnerable directory  with `-w out.csv` and check the file contains correct CSV data.

## Checklist


-   [x] Tests have been added to verify that the new code works (if possible)
-   [ ] Documentation has been updated to reflect changes
-   [x] Changelog has been updated to reflect changes
